docs/source/markdown/podman-push.1.md

```diff
@@ -20,37 +20,30 @@ Images are pushed from those stored in local image storage.
 
 ## DESTINATION
 
- The DESTINATION is a location to store container images
- The Image "DESTINATION" uses a "transport":"details" format.
- If a transport is not given, podman push will attempt to push
- to a registry.
+ DESTINATION is the location the container image is pushed to. It supports all transports from `containers-transports(5)`. If no transport is specified, the `docker` (i.e., container registry) transport is used.  For remote clients, `docker` is the only supported transport.
 
- Multiple transports are supported:
-
-  **dir:**_path_
-  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
-
-    $ podman push myimage dir:/tmp/myimage
-
-  **docker://**_docker-reference_
-  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`. If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
-
-    $ podman push myimage quay.io/username/myimage
+```
+# Push to a container registry
+$ podman push quay.io/podman/stable
 
-  **docker-archive:**_path_[**:**_docker-reference_]
-  An image is stored in the `docker save` formatted file.  _docker-reference_ is only used when creating such a file, and it must not contain a digest.
+# Push to a container registry via the docker-transport
+$ podman push docker://quay.io/podman/stable
 
-    $ podman push myimage docker-archive:/tmp/myimage
+# Push to a container registry with another tag
```

Comment:
```suggestion
# Push to a different repository or tag on a container registry
```
?

Reply:
I can’t see any change.

Reply:
Missed that, sorry.  I was quite in a rush yesterday to get the backports for v3.2 in place.